GroupProject/Pages/Index.cshtml.cs

```diff
@@ -35,24 +35,21 @@ public void OnGet()
 
 
 
-                string Membership = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/MembershipsByCompetition/EPL?key=bc49021bad1943008414c5a75e665961");
```

Comment:
For clarity of exactly what data is being read in, I changed "Membership" to "playerInfoData”